fleet_rent/tests/test_rent_type.py

```diff
@@ -0,0 +1,15 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestRentType(TransactionCase):
+
+    def setUp(self):
+        super(TestRentType, self).setUp()
+
+    def test(self):
+        self.rent_type_obj = self.env['rent.type']
```

Comment:
No need extra global variable if you are using only one time object to do the operation ( create, write .. etc)
you can directly use  self.env['rent.type'].create